bin/dfx-sns-disburse-neuron

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+SOURCE_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+PATH="$SOURCE_DIR:$PATH"
+. "$SOURCE_DIR/versions.bash"
```

Comment:
If you define a `print_help(){}` function, you can include a description.  At present `--help` will describe the arguments but not what the script does.  For an example, please see: https://github.com/dfinity/nns-dapp/blob/main/scripts/dfx-software-ii-cache#L4

Note: The original `optparse` implementation did not  support descriptions so most of the commands don't have such a function, however I think the descriptions are helpful and would like to see all the help texts describe what the corresponding program does.

Reply:
Support for `print_help()` was not in the original optparse, hence the low density of adequate help messages.